index.md

```diff
@@ -62,22 +62,23 @@ Who ACT is **not** for: Users of accessibility tools (web developers, content au
 
 ## Status of ACT Rules Format 1.0
 
-[Accessibility Conformance Testing (ACT) Rules Format 1.0](https://www.w3.org/TR/act-rules-format/) is currently a Working Draft. ACT Rule implementations of this specification are currently in development.
+[Accessibility Conformance Testing (ACT) Rules Format 1.0](https://www.w3.org/TR/act-rules-format/) is currently a stable draft called W3C Proposed Recommendation (see [How WAI Develops Accessibility Standards through the W3C Process](https://www.w3.org/WAI/standards-guidelines/w3c-process/) for more background). A collection of [ACT Rules](https://act-rules.github.io/rules/) conforming to this specification is being developed through the [W3C ACT Rules Community Group](https://www.w3.org/community/act-r/).
 
 ### Technical document format
 
 The ACT Rules Format follows the W3C format for technical specifications, which includes several sections at the beginning: links to different versions, editors, copyright, abstract, and status with the link to errata and the email address for comments. Most WAI specifications have a link at the top to the Table of Contents.
 
 ## Who develops ACT
 
-[ACT Task Force](https://www.w3.org/WAI/GL/task-forces/conformance-testing/) of the Accessibility Guidelines Working Group ([AG WG]( https://www.w3.org/WAI/GL/)) develops the normative ACT Rules Format specification and supporting materials needed to implement the standard.
+The [ACT Task Force](https://www.w3.org/WAI/GL/task-forces/conformance-testing/) of the Accessibility Guidelines Working Group ([AG WG]( https://www.w3.org/WAI/GL/)) develops the normative ACT Rules Format 1.0 specification and supporting materials needed to implement the standard.
 
-The [auto-WCAG Community Group](https://auto-wcag.github.io/auto-wcag/) develops non-normative test rules according to the normative specification. Approved test rules are provided in the [Accessibility Conformance Testing (ACT) Rules Repository](https://w3c.github.io/wcag-act-rules/).
+The [W3C ACT Rules Community Group](https://www.w3.org/community/act-r/) (formerly Auto-WCAG CG) develops non-normative test rules according to the normative specification. Approved test rules will be published in the [Accessibility Conformance Testing (ACT) Rules Repository](https://w3c.github.io/wcag-act-rules/).
 
 Opportunities for contributing to ACT and other WAI work are introduced in [Participating in WAI](/WAI/participation).
 
 ## Additional information
 
 For more information, see:
+* [Harmonizing Accessibility Testing (ACT) post](https://www.w3.org/blog/2019/07/harmonized-accessibility-testing/)
```

Comment:
unless you/we decide to hold off for Rec, per Michael's message

Reply:
need to talk more about this work, not less. more promotion in planning for the REC.